src/features/wireshark/wireshark.ts

```diff
@@ -78,7 +80,7 @@ const locateSharkPathOnLinux = () => {
 export const openInWireshark =
     (pcapPath: string): AppThunk<RootState> =>
     () => {
-        const path = getWiresharkPath();
+        const path = getWiresharkPath() as string;
```

Comment:
I think the `as string` is unnecessary. Maybe you tried to fix something with it and then didn't see that it wasn't really necessary.

```suggestion
        const path = getWiresharkPath();
```